scripts/global_ens/prep/exevs_global_ens_naefs_atmos_prep.sh

```diff
@@ -19,10 +19,10 @@ export run_mpi=${run_mpi:-'yes'}
 export CLIMO=$FIXevs/climos/atmos
 export MASKS=$FIXevs/mask
 
-export COMOUT_gefs_bc=${COMOUT}.${INITDATE}/gefs_bc
-export COMOUT_cmce_bc=${COMOUT}.${INITDATE}/cmce_bc
-mkdir -p $COMOUT_gefs_bc
-mkdir -p $COMOUT_cmce_bc
+#export COMOUT_gefs_bc=${COMOUT}.${INITDATE}/gefs_bc
+#export COMOUT_cmce_bc=${COMOUT}.${INITDATE}/cmce_bc
+#mkdir -p $COMOUT_gefs_bc
+#mkdir -p $COMOUT_cmce_bc
```

Comment:
Let's remove lines 22-25 now that they are loaded in the prep J-job.

Reply:
Done!